scripts/dss_subscription.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import requests
+
+import hca
+from matrix.common.config import MatrixInfraConfig
+
+if __name__ == '__main__':
+    gcp_service_acct_creds = json.loads(base64.b64decode(MatrixInfraConfg().gcp_service_acct_creds).decode())
+    with open('gcp_creds.json', 'w') as outfile:
+        json.dump(data, outfile)
+    os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = os.getcwd() + "/gcp_creds.json"
+    deployment_stage = os.environ['DEPLOYMENT_STAGE']
+    replica = "aws"
+    jmespath_query = "(event_type==`CREATE` || event_type==`TOMBSTONE` || event_type==`DELETE`) \
+        && (files.library_preparation_protocol[].library_construction_approach.ontology==`EFO:0008931` \
+        || files.library_preparation_protocol[].library_construction_approach.ontology_label==`10X v2 sequencing`) \
+        && files.analysis_process[].process_type.text==`analysis`"
+
+    if deployment_stage == "prod":
+        swagger_url = "https://dss.data.humancellatlas.org/v1/swagger.json"
+        matrix_callback = "https://matrix.data.humancellatlas.org/v0/dss/notifications"
+    else:
+        swagger_url = f"https://dss.{deployment_stage}.data.humancellatlas.org/v1/swagger.json"
+        matrix_callback = f"https://matrix.{deployment_stage}.data.humancellatlas.org/v0/dss/notifications"
+
+    dss_client = hca.dss.DSSClient(swagger_url=swagger_url)
```

Comment:
Would you like to or should I later re-use `get_dss_client` here?

Reply:
Oh I see you're also setting the matrix callback. I would still opt for re-use - I'll do it